client/package.json

```diff
@@ -9,6 +9,7 @@
         "@material-ui/icons": "^4.9.1",
         "@material-ui/lab": "^4.0.0-alpha.56",
         "@material-ui/pickers": "^3.3.10",
+        "antalmanac": "file:..",
```

Comment:
Did you add this manually or was it generated by running `npm install`? Not sure what this does or if it's necessary. @LawrenceDavidLu

Reply:
Raman and I are discussing this on Discord. Looks like it's from the newer version of npm.

https://discord.com/channels/772739905981644850/780316624020439090/951784755454697532

Reply:
Ok so I don't think this should be included.
`package-lock.json` looks fine otherwise.